staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1600,12 +1600,26 @@ type MigrationConfiguration struct {
 // DeveloperConfiguration holds developer options
 // +k8s:openapi-gen=true
 type DeveloperConfiguration struct {
-	FeatureGates           []string          `json:"featureGates,omitempty"`
-	LessPVCSpaceToleration int               `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
-	MemoryOvercommit       int               `json:"memoryOvercommit,omitempty"`
-	NodeSelectors          map[string]string `json:"nodeSelectors,omitempty"`
-	UseEmulation           bool              `json:"useEmulation,omitempty"`
-	CPUAllocationRatio     int               `json:"cpuAllocationRatio,omitempty"`
+	FeatureGates           []string            `json:"featureGates,omitempty"`
+	LessPVCSpaceToleration int                 `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
+	MemoryOvercommit       int                 `json:"memoryOvercommit,omitempty"`
+	NodeSelectors          map[string]string   `json:"nodeSelectors,omitempty"`
+	UseEmulation           bool                `json:"useEmulation,omitempty"`
+	CPUAllocationRatio     int                 `json:"cpuAllocationRatio,omitempty"`
+	DebugConfiguration     *DebugConfiguration `json:"debugConfiguration,omitempty"`
+}
+
+// DebugConfiguration sets various components to a debug mode
+// +k8s:openapi-gen=true
+type DebugConfiguration struct {
+	DebugAllVirtComponents bool `json:"debugAllVirtComponents,omitempty"`
```

Comment:
Our logger has no `Debug` concept. We have verbosity levels. I would prefer  if you can set a verbosity level instead of having a boolean here.

Reply:
Yes, I agree. Switching to verbosity levels is better. 
Originally, I thought that `debug` would also mean that we will not destroy the compute container after qemu crash/shutdown.